tomcat/detect.go

```diff
@@ -39,12 +40,22 @@ func (d Detect) Detect(context libcnb.DetectContext) (libcnb.DetectResult, error
 		},
 	}
 
+	m, err := libjvm.NewManifest(context.Application.Path)
+	if err != nil {
+		return libcnb.DetectResult{}, fmt.Errorf("unable to read manifest\n%w", err)
+	}
+
+	if _, ok := m.Get("Main-Class"); ok {
+		return result, nil
```

Comment:
Why are we passing detection in this case, given that `build` will be a noop per https://github.com/paketo-buildpacks/apache-tomcat/pull/36/files#diff-d26c8d2c13b656af64cab6c8204a4ebbed8bcba25ab45450e856a32b1a4a22e5R41-R43?

Reply:
Yep, fair enough.  A precompiled artifact that has a `Main-Class` is the only one of the four possible combinations that _can_ definitively case detection failure.  I’ll update.